### DISCUSSION TITLE
Will AlDente still work with application closed?

### DISCUSSION THREAD

**Discussion by Jseeca** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#161] [Q&A]
So if you were to uninstall it the application, your recommended to set back the percentage to 100% as the charge will still be limited. So because of that it should be fine for me to quit the application and still have the charging limit? I just want have less background processors even if it's minimal haha.

**Comment by MatthiasKerbl** [comment]
Hi @Jseeca,
if you are using AlDente on an Intel MacBook with the feature "Use Classic SMC Key (Intel)" activated, you can close the app and your charging will still be limited. However, if you are on an Apple Silicon MacBook this is not possible.

**Reply by Jseeca** [reply]
I see, thanks for the information!
